app/App/Account/Account/Balances/helpers.js

```diff
@@ -36,3 +36,14 @@ export function balance (rawBalance, quote = {}) {
     displayValue: totalValue.isZero() ? '0' : formatUsdRate(totalValue, 0)
   }
 }
+
+export const sortByTotalValue = (a, b) => {
```

Comment:
can we move this whole file to something like `./resources/domain/balance` or something consistent with our other domain-specific functions?